android/src/main/java/com/oney/WebRTCModule/PeerConnectionObserver.java

```diff
@@ -452,9 +362,154 @@ public void onSignalingChange(PeerConnection.SignalingState signalingState) {
         webRTCModule.sendEvent("peerConnectionSignalingStateChanged", params);
     }
 
+    /**
+     * Triggered when a new track is signaled by the remote peer, as a result of
+     * setRemoteDescription.
+     */
     @Override
     public void onAddTrack(final RtpReceiver receiver, final MediaStream[] mediaStreams) {
         Log.d(TAG, "onAddTrack");
+
+        // Get The timestamp so that we can reorder the transceivers on the JS layer
+        Long tsLong = System.currentTimeMillis()/1000;
+        String ts = tsLong.toString();
+
+        MediaStreamTrack track = receiver.track();
+
+        if (track == null) {
```

Comment:
This should be impossible here, since the receiver is live.

Reply:
Removed.